.github/workflows/env-setup-delete-redis.yml

```diff
@@ -0,0 +1,36 @@
+name: Env Setup - Delete Redis in Openshift
+run-name: Env Setup - Delete Redis in Openshift from ${{ inputs.environment }} using ${{ inputs.gitRef }}
+
+concurrency: redis-setup
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: "Environment"
+        required: true
+        type: environment
+      gitRef:
+        description: "Build Ref"
+        required: true
+        default: ""
+
+jobs:
+  deleteRedis:
+    name: Delete Redis in Openshift
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.environment }}
+    env:
+      NAMESPACE: ${{ secrets.OPENSHIFT_ENV_NAMESPACE }}
+    steps:
+      - name: Checkout Target Branch
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.ref_name }}
+      - name: Log in to OpenShift
+        run: |
+          oc login --token=${{ secrets.SA_TOKEN }} --server=${{ vars.OPENSHIFT_CLUSTER_URL }}
+      - name: Delete Redis
+        working-directory: "./devops/"
+        run: |
+          make delete-redis NAMESPACE=${{ secrets.OPENSHIFT_ENV_NAMESPACE }}
```

Comment:
As the namespace is already defined in the env we can use, `$NAMESPACE` instead of looking up secret.

Reply:
As it is in the env we don't need to pass it as an argument. Removed.